orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -5229,6 +5229,36 @@ paths:
       summary: Modify a rating group
       tags:
       - Rating Groups
+  /networks/{network_id}/sentry:
+    get:
+      parameters:
+      - $ref: '#/parameters/network_id'
+      responses:
+        "200":
+          description: Network-wide Sentry.io configuration
+          schema:
+            $ref: '#/definitions/network_sentry_config'
+        default:
+          $ref: '#/responses/UnexpectedError'
+      summary: Get the Sentry.io configuration for network
+      tags:
+      - Networks
```

Comment:
is the linter warning just wrong? or does yaml not care about indententation?

Reply:
That's interesting... This is a generated file. In the manually edited `orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go`, list items are indented correctly. But this generated has file has unindented lists throughout. It's a `combine_swagger` issue I think.

Reply:
This file is generated. The output is correct, yaml is weird